Update CrowdStrikeFalcon.yml (#34916)
demisto/content: Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/README.md

3. Optional: You can go to one of the *fetch query* parameters and select the query to fetch the events from CrowdStrike Falcon.
4. In the *Mirroring Direction* integration parameter, select in which direction the incidents should be mirrored:

    | **Option** | **Description** |
    | --- | --- |
    | None | Turns off incident mirroring. |
    | Incoming | Any changes in CrowdStrike Falcon events (mirroring incoming fields) will be reflected in Cortex XSOAR incidents. |
    | Outgoing | Any changes in Cortex XSOAR incidents will be reflected in CrowdStrike Falcon events (outgoing mirrored fields). |
    | Incoming And Outgoing | Changes in Cortex XSOAR incidents and CrowdStrike Falcon events will be reflected in both directions. |

5. Optional: Check the *Close Mirrored XSOAR Incident* integration parameter to close the Cortex XSOAR incident when the corresponding event is closed in CrowdStrike Falcon.
6. Optional: Check the *Close Mirrored CrowdStrike Falcon Incident or Detection* integration parameter to close the CrowdStrike Falcon incident or detection when the corresponding Cortex XSOAR incident is closed.

> Newly fetched XSOAR incidents will be mirrored in the chosen direction. However, this selection does not affect existing incidents.

Author: ```suggestion
Newly fetched Cortex XSOAR incidents will be mirrored in the chosen direction. However, this selection does not affect existing incidents.
```

> Exmample: `cloud_provider: 'aws'+account_id: 'my_id'`

Author: ```suggestion
Example: `cloud_provider: 'aws'+account_id: 'my_id'`
```

> Exmample: `cloud_provider=aws&region=eu-west-2`

Author: ```suggestion
Example: `cloud_provider=aws&region=eu-west-2`
```

>| runscript | runscript -Raw=Write-Output 'Hello, World! | a1a1a1a1a1a1a1a1a1a1a1a1a1a1a1a1 |  | Hello, World! |                                    Type         Size (bytes)    Size (MB)       Last Modified (UTC-5)     Created (UTC-5)          <br/>----                                     ----         ------------    ---------       ---------------------     ---------------          <br/>$Recycle.Bin                             &lt;Directory&gt;  --              --              11/27/2018 10:54:44 AM    9/15/2017 3:33:40 AM     <br/>ITAYDI                                   &lt;Directory&gt;  --              --              11/19/2018 1:31:42 PM     11/19/2018 1:31:42 PM     |

### cs-falcon-run-get-command

***

> Batch executes 'get' command across hosts to retrieve files.

Author: "The running status you requested in the get command can be checked with cs-falcon-status-get-command."

            },
            "host_info": {
                "hostname": "host",
                "local_ip": "10.128.0.7",
                "machine_domain": "",
                "os_version": "version",
                "ou": "",
                "site_name": "",
                "system_manufacturer": "manufactor",
                "tags": [],
                "platform": "Windows",
                "instance_id": "instance id",
                "service_provider_account_id": "id",
                "service_provider": "id",
                "os_build": "os build",
                "product_type_desc": "Server"
            },
            "cve": {
                "id": "CVE-20212-2222"
            }
        }
    
}
```

#### Human Readable Output

| CVE ID | Host Info hostname | Host Info os Version | Host Info Product Type Desc | Host Info Local IP | Host Info ou | Host Info Machine Domain | Host Info Site Name | CVE Exploitability Score | CVE Vector |
| --- | --- | --- | --- |  --- | --- |  --- | --- |  --- | --- |
| CVE-20212-2222 |  host | 1 | Server | ip |  |  | site | 5.5 |  |

## Troubleshooting
When encountering connectivity or authorization errors within Cortex XSOAR 8, 

> it necessary to include the IP corresponding to the relevant region into the CrowdStrike Falcon allow list.

Author: ```suggestion
it is necessary to include the IP address corresponding to the relevant region in the CrowdStrike Falcon allow list.
```
